packages/react-components/react-button/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -208,6 +208,23 @@ const useRootStyles = makeStyles({
         color: tokens.colorNeutralForeground2BrandPressed,
       },
     },
+
+    '@media (forced-colors: active)': {
+      ':hover': {
+        color: 'canvas',
```

Comment:
_nit:_ You should use capitalized "Canvas" for consistency's sake

```suggestion
        color: 'Canvas',
```

Ditto elsewhere in your changes.